include/nccl_ofi_rdma.h

```diff
@@ -730,12 +734,12 @@ struct nccl_net_ofi_rdma_ep {
 	/* Pending requests queue */
 	nccl_ofi_deque_t *pending_reqs_queue;
 
-	/* Free list of bounce buffers */
-	nccl_ofi_freelist_t *bounce_buff_fl;
+	/* Free list of bounce buffers (ctrl rail) */
+	nccl_ofi_freelist_t *bounce_buff_ctrl_fl;
```

Comment:
These are not bounce buffers.  They're rx buffers for control messages.

Reply:
Changed one occurrence of this. We need to rename `bounce` to `recv` more generally in the plugin, but would prefer to do this in a separate PR.